scripts/release/util.py

```diff
@@ -0,0 +1,77 @@
+import os
+import sys
+from typing import Any
+from typing import Callable
+from typing import Optional
+from typing import Sequence
+
+import click
```

Comment:
Why are we using click?

Reply:
For `echo` mostly, but also if we want to add flags or options later it's easier than optparse.

I'm not really worried about dependency chain here since it's just for internal use.

Reply:
Yeah sounds good